mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -517,20 +411,20 @@ private Record withIdentityPartitionColumns(
   }
 
   private static Table findTable(Configuration conf) {
-    String path = conf.get(TABLE_PATH);
+    String path = conf.get(InputFormatConfig.TABLE_PATH);
     Preconditions.checkArgument(path != null, "Table path should not be null");
     if (path.contains("/")) {
       HadoopTables tables = new HadoopTables(conf);
       return tables.load(path);
     }
 
-    String catalogFuncClass = conf.get(CATALOG);
+    String catalogFuncClass = conf.get(InputFormatConfig.CATALOG);
     if (catalogFuncClass != null) {
       Function<Configuration, Catalog> catalogFunc = (Function<Configuration, Catalog>)
-          DynConstructors.builder(Function.class)
-                         .impl(catalogFuncClass)
-                         .build()
-                         .newInstance();
+              DynConstructors.builder(Function.class)
+                      .impl(catalogFuncClass)
+                      .build()
+                      .newInstance();
```

Comment:
Can you revert the whitespace-only changes in this file? Otherwise this is going to have more conflicts than necessary.

Reply:
Yes, definitely.